readme.md

```diff
@@ -5,3 +5,22 @@ Declares the Airbyte Protocol.
 ## Key Files
 * `airbyte_protocol.yaml` - declares the Airbyte Protocol (in JSONSchema)
 * `io.airbyte.protocol.models` - this package contains various java helpers for working with the protocol.
+
+
+## Pull Requests Titles Must Conform to Convential Commits Convention
+We are leveraging the [Release Please Action](https://github.com/marketplace/actions/release-please-action) to manage our version bumping and releasing.
+This action relies on the use of the [convential commits convention](https://www.conventionalcommits.org/en/v1.0.0/) to determine whether to bump major, minor, or patch versions. Since we use squash merging, the only commits we see in our history are the titles of our pull requests. This is why we are following conventional commits for our pull request titling. Your actual commits do not need to follow this convention because they never show up in the git history.
+
+Here is a summary of what Release Please looks for in your pull request title.
+
+> The most important prefixes you should have in mind are:
+>
```

Comment:
what will happen if i do a different conventional commit other than these three? Will it always make a patch version?

Reply:
I’m certain that the automation will not run to generate a new release candidate. I’m not certain if the PRs that contain `chore:` or `ci:` are included in the changelog.

Reply:
I’ve verified that `chore:` will not show up in the pr/changelog. Is this something we would desire?

Reply:
There is a setting to allow other conventional commit labels to show up. https://github.com/marketplace/actions/release-please-action#overriding-the-changelog-sections

Reply:
With the addition of https://github.com/airbytehq/airbyte-protocol/pull/10/files#diff-87db21a973eed4fef5f32b267aa60fcee5cbdf03c67fafdc2a9b553bb0b15f34R18-R46, I've verified that `chore`, `ci`, 
 and `docs` will trigger a patch bump.